pkg/terraform/install.go

```diff
@@ -26,13 +35,49 @@ func (m *Mixin) Install() error {
 		return err
 	}
 
-	var step InstallStep
-	err = yaml.Unmarshal(payload, &step)
+	var action InstallAction
+	err = yaml.Unmarshal(payload, &action)
 	if err != nil {
 		return err
 	}
+	if len(action.Steps) != 1 {
+		return fmt.Errorf("expected a single step, but got %d", len(action.Steps))
+	}
+	step := action.Steps[0]
+
+	if step.LogLevel != "" {
```

Comment:
If they specify `--debug` I think we should turn up the log level up a bit too. What do you think?

Reply:
You know, that's what I started with (setting TF_LOG to TRACE or DEBUG when --debug was set), but noticed it can get quite verbose (so, crowd our CI logs, etc.) . But, I do agree, it does kinda make sense.  I'm up for whatevs.

Reply:
Ok yeah ignore my suggestion, that's a big firehose you can't turn off on second thought. Carry on!